sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/models/CosmosEncryptionType.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.encryption.models;
+
+/**
+ * Algorithms for use with client-side encryption support in Azure Cosmos DB.
+ */
+public class CosmosEncryptionType {
```

Comment:
>ublic  [](start = 1, length = 6)

We got rid of this from the public surface on .net so people can just use the constants from the AAP library

Reply:
Its better to give cx CosmosEncryptionType in cosmos project , its discovery will be easy instead of depending on AAP lib, I am using it explicitly in my customer samples  . However if it need to be removed , we will revisit it after preview